samples/builtin-kernel-example/builtin-kernel-example.cpp

```diff
@@ -0,0 +1,119 @@
+/***************************************************************************
+ *
+ *  Copyright (C) 2016 Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  Codeplay's ComputeCpp SDK
+ *
+ *  builtin_kernel_example.cpp
+ *
+ *  Description:
+ *    Example of using an OpenCL builtin kernel with SYCL via the codeplay
+ *    extension
+ *
+ **************************************************************************/
+
+#include <CL/sycl.hpp>
+
+#include <array>
+#include <iostream>
+
+int main() {
+
+  queue testQueue(TEST_SELECTOR, TEST_ASYNC_HANDLER);
```

Comment:
This is missing `cl::sycl::`, or `using namespace` elsewhere in the file. I think for a sample, either is fine.
This is also using the TEST_SELECTOR stuff, which isn't available in the SDK - probably suffices to replace with a default_selector and a handler that simply throws the first exception, with optional output.

Reply:
That was the wrong commit actually, updated now.